osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -100,7 +100,8 @@ def extract_excerpts(execution_configuration, extraction_order, bbox_args, conve
                 if len(os.listdir(settings.RESULT_MEDIA_ROOT)) > 0:
                     for result_file_name in os.listdir(settings.RESULT_MEDIA_ROOT):
                         # gis files are packaged in a zip file
-                        if GisExcerptConverter.create_output_file(extraction_order, result_file_name):
+                        if GisExcerptConverter.create_output_file(
+                                extraction_order, result_file_name, export_format_key):
```

Comment:
This line breaking seems odd to me, can we ie. extract this into a function, which the name describes what it does and get rid of the line-break? (Maybe there's a better way, just an idea).